src/scenes/App/index.js

```diff
@@ -46,7 +44,7 @@ export default class App extends Component {
         <WalletUnlockDialog />
         <ErrorDialog />
         <TxConfirmDialog txDesc="hey" txAmount={10} txToken="QTUM" />
-        <TutorialCarouselDialog />
+        <TutorialCarouselDialog langHandler={langHandler} lang={lang} />
```

Comment:
don't need to pass `lang` here. We can access this in `intl.locale`

Reply:
done